cmd/indexer/root.go

```diff
@@ -170,6 +171,7 @@ func init() {
 	// Syncing related flags
 	ServerCmd.Flags().Int64(flags.SyncTargetBlock, 0, "The block number to sync to initially")
 	ServerCmd.Flags().Int64(flags.SyncFromBlock, 0, "The init block number to sync to initially")
+	ServerCmd.Flags().BoolVar(&onlySubscribe, flags.SyncOnlySubscribe, true, "Enable to only index subscribed accounts")
```

Comment:
Why not to use `ServerCmd.Flags().Bool`?

Reply:
make up a style uniform.

Reply:
OK